lib/views/file-patch-header-view.js

```diff
@@ -40,8 +46,31 @@ export default class FilePatchHeaderView extends React.Component {
   }
 
   renderTitle() {
-    const status = this.props.stagingStatus;
-    return `${status[0].toUpperCase()}${status.slice(1)} Changes for ${this.props.relPath}`;
+    if (this.props.itemType === ChangedFileItem) {
+      const status = this.props.stagingStatus;
+      return (
+        <span>{status[0].toUpperCase()}{status.slice(1)} Changes for {this.renderPath()}</span>
+      );
+    } else {
+      return this.renderPath();
+    }
+  }
+
+  renderPath() {
+    const dirname = path.dirname(this.props.relPath);
+    const basename = path.basename(this.props.relPath);
+
+    if (dirname === '.') {
+      return <span className="gitub-FilePatchHeaderView-basename">{basename}</span>;
+    } else {
+      // TODO: double check that the forward-slash works cross-platform...
```

Comment:
🚨 Unresolved TODO

Reply:
I'm glad I checked this! It turns out that while git itself always uses forward slashes, we explicitly translate paths in patches with `toNativePathSep`:

<img width="511" alt="whoops" src="https://user-images.githubusercontent.com/17565/48591195-d5c4e780-e910-11e8-84dc-28e561c2b482.PNG">

Fortunately the fix is to just use `path.sep` instead:

<img width="546" alt="new-and-improved" src="https://user-images.githubusercontent.com/17565/48591206-e07f7c80-e910-11e8-8454-0cafc1f0684c.PNG">